modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= AWS S3 compatible backup storage providers
+
+OADP supports a variety of object storage providers for different backup and snapshot operations.
+
+== AWS S3 drivers for backup storage locations
+
+=== Supported
```

Comment:
I looked a little and couldn't find an example of `Supported` being used as a heading. I think reorganizing the headings and content might be useful here.